sources/api/schnauzer/src/helpers.rs

```diff
@@ -782,59 +731,6 @@ pub fn ecr_prefix(
     Ok(())
 }
 
-/// The `pause-prefix` helper is used to map an AWS region to the correct pause
-/// container registry.
-///
-/// Because the repo URL includes the the registry number, we created this helper
-/// to lookup the correct registry number for a given region.
-///
-/// This helper takes the AWS region as its only parameter, and returns the
-/// fully qualified domain name to the correct registry.
-///
-/// # Fallback
-///
-/// If we do not have the region in our map, a fallback region and registry number
-/// are returned.  This would allow a version of Bottlerocket to run in a new region
-/// before this map has been updated.
-///
-/// # Example
-///
-/// In this example the registry number for the region will be returned.
-/// `{{ pause-prefix settings.aws.region }}`
-///
-/// This would result in something like:
-/// `602401143452.dkr.ecr.eu-central-1.amazonaws.com`
-pub fn pause_prefix(
-    helper: &Helper<'_, '_>,
-    _: &Handlebars,
-    _: &Context,
-    renderctx: &mut RenderContext<'_, '_>,
-    out: &mut dyn Output,
```

Comment:
`schnauzer` is currently also located in the `bottlerocket` repo to facilitate template-based migrations.

We should probably also remove this code there, and make the requisite updates for `ap-southeast-5` so that migrations do the right thing.